src/components/stepforms/AppendStepForm.vue

```diff
@@ -5,10 +5,13 @@
       class="pipelinesInput"
       v-model="editedStep.pipelines"
       name="Select datasets to append:"
-      :options="[...availablePipelines, ...domains]"
+      :options="options"
       placeholder="Select datasets"
       data-path=".pipelines"
       :errors="errors"
+      track-by="trackBy"
+      label="label"
+      with-example
```

Comment:
What does the with-example directive is used for ? Did'nt find it in vue-multiselect documentation ?